server/public/styles.css

```diff
@@ -0,0 +1,107 @@
+* {
+  box-sizing: border-box;
+  font-family: 'Roboto', sans-serif;
+}
+
+header, footer, main {
+  max-width: 1024px;
+  margin: 0 auto;
+}
```

Comment:
Since these are all stacked on one another, you can wrap them in a single element that is styled this way. Actually, Bootstrap already has a `.container` class for this purpose.